insights/parsers/tests/test_ps.py

```diff
@@ -29,17 +29,58 @@
 nginx    111435 111434  0 22:32 ?        00:00:00 nginx: worker process
 """
 
+PsAex_TEST_DOC = """
```

Comment:
This test data should not be here?

Reply:
@xiangce thanks for catching that, I will remove it.